packages/application-config/client.d.ts

```diff
@@ -1,5 +1,13 @@
 type CSSModuleClasses = { readonly [key: string]: string };
 
+declare namespace NodeJS {
+  // Used for global variable replacements.
+  interface ProcessEnv {
+    readonly DEBUG: 'true' | 'false';
+    readonly NODE_ENV: 'development' | 'production' | 'test';
```

Comment:
Don't we assign a value to this somewhere? Maybe then it can't be readonly.

Reply:
Yes right, let me remove it then as it's then more confusing.